addons/crm_livechat/models/chatbot_script_step.py

```diff
@@ -1,23 +1,35 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import _, models, fields
+from ast import literal_eval
+
+from odoo import models, fields
+from odoo.osv import expression
 
 
 class ChatbotScriptStep(models.Model):
     _inherit = 'chatbot.script.step'
 
     step_type = fields.Selection(
-        selection_add=[('create_lead', 'Create Lead')], ondelete={'create_lead': 'cascade'})
+        selection_add=[
+            ("create_lead", "Create Lead"),
+            ("create_lead_and_forward", "Create Lead & Forward"),
+        ],
+        ondelete={"create_lead": "cascade", "create_lead_and_forward": "cascade"},
+    )
     crm_team_id = fields.Many2one(
-        'crm.team', string='Sales Team', ondelete='set null',
+        'crm.team', string='Sales Team', ondelete='set null', index="btree_not_null",
```

Comment:
Why do you have an index ? Not sure we are going to have thousands of steps, do we ?